src/app/new-filters/n-filter.component.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../assets/stylesheets/_base';
+.filter-pf {
+  a { cursor: pointer; }
+}
+.dropdown-menu { min-width: 176px; }
```

Comment:
with view encapsulation turned off, has it been verified that this change to the width of the class `.dropdown-menu` will not negatively impact the other dropdown menus on any page that this widget is included on?